File: java/BillsApi/src/main/java/com/apilangarena/javabillsapi/application/bills/create/CreateBillResult.java
package com.apilangarena.javabillsapi.application.bills.create;

import java.math.BigDecimal;
import java.time.LocalDate;

public record CreateBillResult(
    Long id,
    String billNumber,
    LocalDate issuedAt,
    BigDecimal subtotal,
    BigDecimal tax,
    BigDecimal total,
    String currency
) {
}

File: kotlin/BillsApi/src/main/kotlin/com/apilangarena/billsapi/application/bills/create/CreateBillResult.kt
package com.apilangarena.billsapi.application.bills.create

import java.math.BigDecimal
import java.time.LocalDate

data class CreateBillResult(
    val id: Long,
    val billNumber: String,
    val issuedAt: LocalDate,
    val subtotal: BigDecimal,
    val tax: BigDecimal,
    val total: BigDecimal,
    val currency: String,
)
